test/e2e/README.md

```diff
@@ -5,18 +5,52 @@ libsinsp. Said tests are based around 2 containers:
 - One running the `sinsp-example` binary
 - Another one running the actual tests and verifying their outcome.
 
-## sinsp container
+## Running the tests
+The e2e tests use docker to run a few containers and execute commands in them
+making it relatively easy to ensure events are the one we are looking for by
+checking the `container.id` field in events. As the runner, pytest is used for
+its flexibility and ease of configuration and a Python module is used to hold
+common helper functions.
+
+### Install Python dependencies
```

Comment:
Maybe totally personal preference, but I could see us just suggesting to install all python dependencies in a virtual env and refer to some tutorial instead of explaining here aka we could remove all venv instructions.

Reply:
Yeah, that might be better, I'll take another spin on these instructions.